client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/entities/Schema.java

```diff
@@ -16,13 +16,23 @@
 
 package io.confluent.kafka.schemaregistry.client.rest.entities;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.util.StdConverter;
 
+import javax.ws.rs.DefaultValue;
+
+import io.confluent.kafka.schemaregistry.avro.AvroSchema;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class Schema implements Comparable<Schema> {
 
   private String subject;
   private Integer version;
   private Integer id;
+  private String schemaType = AvroSchema.AVRO;
```

Comment:
Should we make `schemaType` an enum type?

Reply:
The user can provide a custom `SchemaProvider`, so it should remain a string to remain open since an enum is closed.